addons/portal/data/mail_templates.xml

```diff
@@ -3,13 +3,20 @@
 
     <template id="portal_share_template">
         <div>
+            <br/>
             <p>Dear <span t-esc="partner.name"/>,</p>
-            <p>You have been invited to access the following document:</p>
+            <p><span t-out="record.env.user.name"/> has invited you to access the following <span t-out="record._description.lower()"/>:</p>
             <br/>
             <a t-attf-href="#{share_link}" style="background-color: #875A7B; padding: 10px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 12px;"><strong>Open </strong><strong t-esc="record.display_name"/></a><br/>
             <br/>
             <p t-if="note" style="white-space: pre-wrap;" t-esc="note"/>
         </div>
+        <div style="font-size: 13px; padding: 30px 0;">
+            <div>Best regards,</div>
+            <div style="font-size: 13px;">
+                <span t-out="record.env.user.sudo().signature"/>
```

Comment:
Do we always have a signature or should we fallback on the name?

Reply:
As it is used in a message_post, you could instead force a "email_add_signature" instead and fallback on classic usage.

Reply:
@tde-banana-odoo , @IT-Ideas 
we have used `mail_notification_light` notification layout and it does not support the option `email_add_signature`. 
either we can add the generic signature in the main notification layout(`mail_notification_light`) or we can add it over here for a specific case. what do you suggest?

Thanks

Reply:
After discussing with Alexandra, we revert that change. 🙂